ruleset.xml

```diff
@@ -8,9 +8,24 @@
   </description>
   <rule ref="rulesets/java/basic.xml"/>
   <rule ref="rulesets/java/braces.xml"/>
+  <rule ref="rulesets/java/design.xml">
+    <exclude name="UseUtilityClass"/>
```

Comment:
Requires utility classes with only static method to have private constructor and be made final.
https://pmd.github.io/pmd-5.8.1/pmd-java/rules/java/design.html#UseUtilityClass